server/handlers.go

```diff
@@ -94,7 +94,7 @@ func initHTMLTemplates() *html_template.Template {
 }
 
 func healthHandler(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprintf(w, "Approaching Neutral Zone, all systems normal and functioning.")
+	_, _ = w.Write([]byte("Approaching Neutral Zone, all systems normal and functioning."))
```

Comment:
what' the ration behind this change?

Reply:
According to the docs i found it is recommended to directly use the Write Method. I am ok changing this back tho.

Reply:
I don't have a strong opinione, but seems that the previous doesn't need error handling/ignoring

I would rather replace with `fmt.Fprintln`, unless we actually have any print formatting

Reply:
fmt.Fprintln actually required error handling and ignoring, just was omitted until now

Reply:
I should have checked on https://pkg.go.dev/fmt#Fprintf :D 

then it's fine, let's keep `w.Write`

omitting `_, _ = ` maybe?

Reply:
I do plan on hooking up something like `golangci-lint` soon to the repo and at latest then we would need to add it :D